galgebra/mv.py

```diff
@@ -33,6 +33,82 @@
 
 half = Rational(1, 2)
 
+
+# Add custom settings to the builtin latex printer
+_LatexPrinter._default_settings.update({
+    'galgebra_mv_fmt': 1
+})
+_StrPrinter._default_settings.update({
+    'galgebra_mv_fmt': 1
+})
+
+
+class _FmtResult:
```

Comment:
Why do we put it in mv.py? I know mv.py is lower in the dependency tree, but it seems should not be in mv.py but a separate file.

Reply:
Because `mv.py` is the only file that uses this `.Fmt()` function, and the `fmt` argument is all about `Mv` terms.